command/issue.go

```diff
@@ -41,6 +41,8 @@ func init() {
 	issueListCmd.Flags().StringP("state", "s", "open", "Filter by state: {open|closed|all}")
 	issueListCmd.Flags().IntP("limit", "L", 30, "Maximum number of issues to fetch")
 	issueListCmd.Flags().StringP("author", "A", "", "Filter by author")
+	issueListCmd.Flags().StringP("mentioned", "", "", "Filter by mention")
```

Comment:
I'm not 100% sold on "mentioned" since it's past tense. How about present tense, e.g. `--mentions ampinsk`? @ampinsk: would love to know your thougths.

Reply:
Maybe just `--mention`?

Reply:
I have updated the flag name